apps/studio/src/schemas/page.ts

```diff
@@ -3,7 +3,7 @@ import { z } from 'zod'
 const PAGE_LAYOUTS = ['content'] as const
 
 export const getEditPageSchema = z.object({
-  pageId: z.string(),
+  id: z.string(),
```

Comment:
eh, prefer `pageId`, esp when there is another id present

Reply:
sure